src/pseudo/PspUpf.jl

```diff
@@ -2,45 +2,68 @@ using LinearAlgebra
 using Interpolations: linear_interpolation
 using PseudoPotentialIO: load_upf
 
+"""
+Read from file:
```

Comment:
This breaks a bit with the way we handle this in other parts of the code. We can argue what's the better approach though as I'm not sure.

So far we have not actually explained much of the struct fields inside the docstrings of a struct. Rather we did it in in-line comments (check e.g. the Model or the PlaneWaveBasis). I would tend to stick to that for consistency. @azadoks I wonder about your opinion though. You think having this as docstrings adds value for you? @antoine-levitt's argument is the understandable point of avoiding duplicated information (the name of the field is there twice) which can get out of sync.

Reply:
In the Julia documentation, it recommends putting strings above the fields in the struct definition. This looked _really_ weird to me when I tried it, so I scrapped it.
I'm not sure if in-line comments show up when you hover for docs in, e.g. VScode, but both this and the interleaved strings showed up, which was nice.

Many of these explanations are far too long to be on the same line as the field definitions, so even with comments instead of strings, it would be hard to see the code for the comments.

I think in this situation, I'd go back to comments to align with the rest of the code.
In general, I think I could get used to the method from the Julia docs, which has the benefit of never being out of sync and also showing up in editor documentation viewers.

Reply:
The way we do it in Model.jl is that small, four-word explanations go in comments just after the field, and longer comments go to comments just above. Point taken on fancy editors: I don't use any of these, but maybe I'm the dinosaur at this point, so as you wish, as long as it's not too far from the code